docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,104 @@
+# Convert ONNX\* RNN-T Model to the Intermediate Representation (IR) {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction covers conversion of RNN-T model from [MLCommons](https://github.com/mlcommons) repository. Follow 
+the steps below to export a PyTorch\* model into ONNX\* before converting it to IR:
+
+**Step 1**. Clone RNN-T PyTorch implementation from MLCommons repository. Make a shallow clone to pull only RNN-T 
+model without full repository. If you already have a full repository, skip this and go to **Step 2**:
+```bash
+git clone -n https://github.com/mlcommons/inference rnnt_for_openvino --depth 1
+cd rnnt_for_openvino
+git checkout HEAD speech_recognition/rnnt 
```

Comment:
We should specify which particular commit hash we tested our instruction. So I would recommend to hardcode the git commit hash in the command.

Reply:
done